frontend/config-assistant/src/components/toolbar/clearFile.ts

```diff
@@ -22,19 +23,17 @@ export function newEmptyFile(message: string | undefined = undefined): void {
       });
       clearFile();
     },
-    reject: () => {
-      toastService.add({
-        severity: 'error',
-        summary: 'Rejected',
-        detail: 'Config data is kept in File Editor',
-        life: 3000,
-      });
-    },
   });
 }
 export function clearFile() {
+  newEmptyFile('Do you want to clear the File editor?');
   useSessionStore().lastChangeResponsible = ChangeResponsible.Menubar;
   useDataStore().fileData = {};
   useSessionStore().updateCurrentPath([]);
   useSessionStore().updateCurrentSelectedElement([]);
 }
+export function openClearFileDialog() {
+  newEmptySchemafile(
```

Comment:
instead of clearing the schema, you should clear the file. Also move it to the clearSchema.ts and not clearFile.ts